cpg_workflows/jobs/seqr_loader_snps_indels.py

```diff
@@ -0,0 +1,225 @@
+            exclude_intervals_path=exclude_intervals_path,
+            scatter_count=scatter_count,
+            job_attrs=job_attrs,
+            output_prefix=tmp_bucket / f'intervals_{scatter_count}',
+        )
+    if intervals_j:
+        jobs.append(intervals_j)
+
+    # all_output_paths = [out_siteonly_vcf_path]
+    all_output_paths = []
+    if out_siteonly_vcf_part_paths:
+        assert len(out_siteonly_vcf_part_paths) == scatter_count
+        all_output_paths.extend(out_siteonly_vcf_part_paths)
+    if can_reuse(all_output_paths + [to_path(f'{p}.tbi') for p in all_output_paths]):
+        return []
+
+    merged_vcf = b.read_input_group(
+        **{
+            'vcf.gz': str(merged_vcf_path),
+            'vcf.gz.tbi': str(merged_vcf_path) + '.tbi',
+        },
+    )
+
+    split_vcfs_paths = [
+        tmp_bucket / 'split-merged-vcf' / 'parts' / f'part{idx + 1}.vcf.gz' for idx in range(scatter_count)
+    ]
+    siteonly_vcfs: list[hb.ResourceGroup] = []
+
+    split_vcf_j = _add_split_vcf_job(
+        b=b,
+        input_vcf=merged_vcf,
+        intervals=intervals,
+        output_vcf_paths=split_vcfs_paths,
+        job_attrs=(job_attrs or {}) | dict(part='all'),
+    )
+
+    for idx in range(scatter_count):
+        if out_siteonly_vcf_part_paths:
+            siteonly_vcf_path = out_siteonly_vcf_part_paths[idx]
+        else:
+            siteonly_vcf_path = tmp_bucket / 'split-merged-vcf-siteonly' / 'parts' / f'part{idx + 1}.vcf.gz'
+        vcf_part = b.read_input_group(
+            **{
+                'vcf.gz': str(split_vcfs_paths[idx]),
+                'vcf.gz.tbi': str(split_vcfs_paths[idx]) + '.tbi',
+            },
+        )
+        siteonly_j, siteonly_j_vcf = _add_make_sitesonly_job(
+            b=b,
+            input_vcf=vcf_part,
+            output_vcf_path=siteonly_vcf_path,
+            job_attrs=(job_attrs or {}) | dict(part=f'{idx + 1}/{scatter_count}'),
+        )
+
+        siteonly_vcfs.append(siteonly_j_vcf)
+        if siteonly_j:
+            jobs.append(siteonly_j)
+            if split_vcf_j:
+                siteonly_j.depends_on(split_vcf_j)
+
+    jobs = [j for j in jobs if j is not None]
+    for j in jobs:
+        j.name = f'Long Read SNPs Indels Annotation: {j.name}'
+    return jobs
+
+
+def _add_split_vcf_job(
+    b: hb.Batch,
+    input_vcf: hb.ResourceGroup,
+    intervals: list[str] | list[hb.ResourceFile],
+    output_vcf_paths: list[Path],
+    job_attrs: dict | None = None,
+) -> Job:
+    """
+    Split VCF by interval and write to part VCF.
+    Use GATK SelectVariants to split the VCF by interval.
+    """
+    j = b.new_job('SplitVcf', (job_attrs or {}) | {'tool': 'gatk SelectVariants'})
+    j.image(image_path('gatk'))
+    res = STANDARD.set_resources(j, ncpu=2)
+
+    for idx, interval in enumerate(intervals):
+        output_vcf_path = output_vcf_paths[idx]
+        j.declare_resource_group(
+            **{
+                str(idx + 1): {
```

Comment:
why are the outputs/resource groups `idx + 1`? It looks like this would work just as well with plain ol' `idx`